R/functions.R

```diff
@@ -494,9 +492,7 @@ df2vw <- function(data, file_path, namespaces = NULL, keep_space = NULL,
     # namespace load with a yaml file
     if (typeof(namespaces) == "character" && length(namespaces) == 1 &&
         grepl("yaml$", namespaces)) {
-        # requireNamespace(yaml)
-        print("Using YAML file for loading the namespaces")
-        if (requireNamespace("yaml", quiet=TRUE, as.character=TRUE)) {
+        if(requireNamespace("yaml", quietly = TRUE)) {
```

Comment:
Minor note:

```r
stop("The 'yaml' package is needed.", .Call=FALSE)
```

The last parameter needs to be lowercase and the period moved to the end.

```r
call. = FALSE
```

Reply:
This is a typo on my end from the last review. Apologies.